EOLib/Domain/Interact/Guild/GuildCreationRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace EOLib.Domain.Interact.Guild
+{
+    public class GuildCreationRequest
```

Comment:
Look at the other record types (decorated with `[Record]` in EOLib.Domain) - this should follow the immutable pattern

Reply:
Yeah tbh I'm thinking of just ditching this and sending the values as arguments. That sound good?

Reply:
I like the idea of a structure that encapsulates the parameters, but either way is fine if you want to go the other route. Separate parameters is also how the other notifiers do it so it's nice for consistency.